src/components/product-page/BreadCrumbs.vue

```diff
@@ -0,0 +1,28 @@
+<script setup lang="ts">
+  import useCategory from "@/composables/api/useCategory"
+  import { capitalize } from "@/utils"
+
+  defineProps<{ productName: string }>()
+
+  const { category } = useCategory()
+</script>
+
+<template>
+  <nav>
+    <router-link class="link-primary link-animation" :to="{ name: 'home' }">
+      Home
+    </router-link>
+    <router-link class="link-primary link-animation" :to="{ name: 'category' }">
+      {{ capitalize(category?.name) }}
+    </router-link>
+    <span>{{ productName }}</span>
+  </nav>
+</template>
+
+<style scoped lang="postcss">
+  a {
```

Comment:
Please don't use tag as a selector. This is a bad practice. We are in the scoped section, you can just use `link-primary`